static_report/src/components/shared/ColumnCard/ColumnCardHeader.tsx

```diff
@@ -37,8 +36,16 @@ export function ColumnCardHeader({ columnDatum }: Props) {
             boxSize={7}
           />
         </Tooltip>
-        <Text fontWeight={'semibold'} fontSize={'3xl'}>
-          {formatTruncateString(name, 10)}
+        <Text
+          as={'span'}
+          fontSize={'xl'}
+          fontWeight={'semibold'}
+          textOverflow={'ellipsis'}
```

Comment:
If you want to truncate text, you can use `noOfLines` prop

https://chakra-ui.com/docs/components/text/usage#truncate-text